configutil/testplugins/aead/main.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	gkwp "github.com/hashicorp/go-kms-wrapping/plugin/v2"
+	aead "github.com/hashicorp/go-kms-wrapping/v2/aead"
+)
+
+func main() {
+	if err := gkwp.ServePlugin(aead.NewWrapper()); err != nil {
+		fmt.Println("Error serving plugin", err)
+		os.Exit(1)
+	}
+	os.Exit(0)
```

Comment:
Is this necessary?

Reply:
Maybe not. I copied this from elsewhere, where it was copied from elsewhere. I think whoever initially wrote it was trying to be explicit.